<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainW_menu</class>
 <widget class="QMainWindow" name="MainW_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_fecha">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_fecha_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Consolas&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 87 16pt &quot;Arial Black&quot;;
border-radius: 10px; 
padding: 8px 12px;
</string>
    </property>
    <property name="text">
     <string>Menú</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_rfondoblanco">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>260</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.6875, y1:0.216, x2:1, y2:0, stop:1 rgba(255, 255, 255, 255));
border-radius: 9px;
padding: 10px 12px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_reporteventas">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial Black&quot;;
border-radius: 8px;
padding: 6px 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_afondoblanco">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>260</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.6875, y1:0.216, x2:1, y2:0, stop:1 rgba(255, 255, 255, 255));
border-radius: 9px;
padding: 10px 12px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ifondoblanco">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>260</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.6875, y1:0.216, x2:1, y2:0, stop:1 rgba(255, 255, 255, 255));
border-radius: 9px;
padding: 10px 12px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_lfondoblanco">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>260</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.6875, y1:0.216, x2:1, y2:0, stop:1 rgba(255, 255, 255, 255));
border-radius: 9px;
padding: 10px 12px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_actualizarproductos">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial Black&quot;;
border-radius: 8px;
padding: 6px 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_inventario">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial Black&quot;;
border-radius: 8px;
padding: 6px 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_listadistribuidores">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial Black&quot;;
border-radius: 8px;
padding: 6px 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_reporte_ventas">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
border-radius: 10px; 
padding: 8px 12px;
</string>
    </property>
    <property name="text">
     <string>Reporte Ventas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_inventario">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
border-radius: 10px; 
padding: 8px 12px;
</string>
    </property>
    <property name="text">
     <string>Inventario</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_actualizarproductos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
border-radius: 10px; 
padding: 8px 12px;
</string>
    </property>
    <property name="text">
     <string>Actualizar Prouctos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_listadistribuidores">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 115, 255);
color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
border-radius: 10px; 
padding: 8px 12px;
</string>
    </property>
    <property name="text">
     <string>Lista de Distribuidores</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_fecha_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionSistema"/>
    <addaction name="actionClaro"/>
    <addaction name="actionOscuro"/>
   </widget>
   <widget class="QMenu" name="menuM_dulos">
    <property name="title">
     <string>Módulos</string>
    </property>
    <addaction name="actionReporte_Ventas"/>
    <addaction name="actionInventario"/>
    <addaction name="actionActualizar_Productos"/>
    <addaction name="actionLista_de_Dsitribuidores"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuHerramientas"/>
   <addaction name="menuM_dulos"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSistema">
   <property name="text">
    <string>Sistema</string>
   </property>
  </action>
  <action name="actionClaro">
   <property name="text">
    <string>Claro</string>
   </property>
  </action>
  <action name="actionOscuro">
   <property name="text">
    <string>Oscuro</string>
   </property>
  </action>
  <action name="actionReporte_Ventas">
   <property name="text">
    <string>Reporte Ventas</string>
   </property>
  </action>
  <action name="actionInventario">
   <property name="text">
    <string>Inventario</string>
   </property>
  </action>
  <action name="actionActualizar_Productos">
   <property name="text">
    <string>Actualizar Productos</string>
   </property>
  </action>
  <action name="actionLista_de_Dsitribuidores">
   <property name="text">
    <string>Lista de Dsitribuidores</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
